.github/workflows/build.yml

```diff
@@ -39,8 +39,6 @@ jobs:
           version: '0.11.6'
         - name: cargo-fuzz
           version: '0.12.0'
-        - name: soroban-cli
-          version: '20.0.0-rc.4.1'
```

Comment:
soroban-cli fails for the same reason as cargo-workspaces, but this is old, we don't need it anymore.